scripts/test.sh

```diff
@@ -38,18 +37,9 @@ while [[ $# -gt 0 ]]; do
       shift
       shift
       ;;
-    -l|--lint)
-      lint=true
-      shift
-      ;;
-    -t|--tools)
-      tools=true
-      shift
-      ;;
```

Comment:
@kakkoyun QQ: what's replacing the `lint` command?

Reply:
`make lint` and there is another `scripts/lint.sh` is coming in my reverted PR.